phoenix/slackbot/auth.py

```diff
@@ -16,8 +16,6 @@ def authenticate(self, request):
         if not slack_user_id and "payload" in request.data:
             data = json.loads(request.data["payload"])
             slack_user_id = data["user"]["id"]
-        else:
-            logger.warning("Unable to retrieve slack user")
```

Comment:
I don't think this is the right solution to this problem. The readiness and liveness probe views shouldn't need authentication at all. I'd rather fix that instead of just removing this logging, which doesn't seem that useless imo.

Reply:
OK, removed authentication from the probe views.